index.js

```diff
@@ -19,6 +24,17 @@ module.exports = {
     app.options.sassOptions.includePaths.push(
       'node_modules/@appuniversum/appuniversum'
     );
+
+    let addonOptions = app.options[this.name];
+    if (addonOptions && addonOptions.dutchDatePickerLocalization) {
+      this.options[
+        '@embroider/macros'
+      ].setOwnConfig.dutchDatePickerLocalization = true;
+    } else {
+      this.ui.writeDeprecateLine(
```

Comment:
This will show a warning in terminal when building the app since it's possible to miss the browser deprecation. Is that overkill?

Reply:
I've removed it again, not all projects might be using the component so it would only be unnecessary noise.